### PR TITLE
some improvements on the strings.

### PR DESCRIPTION
**Pull request by Raymondqx**

### CODE REVIEW

**Review by FnrDev — APPROVED** [review]

File: lyra/bot.json [hunk 1]
```diff
@@ -210,36 +210,34 @@
...
-    "addadmin": "Add admin to your bot 'You can use the main bot to add the admin to all bots'",
-    "deladmin": "Remove admin from your bot 'You can use the main bot to remove the admin from all bots'",
-    "addserver": "Allow the bot enter the server 'You can use the main bot to allow the server to all bots'",
-    "connect": "Make all of your bots data like this bot except the prefix",
-    "vip": "Show subscription end date"
+    "addadmin": "Add admin to your bot 'You can use the main bot instead to add the admin to all bots'",
+    "deladmin": "Remove admin from your bot 'You can use the main bot instead to remove the admin from all bots'",
+    "addserver": "Give bot access to join multiple servers 'You can use the main bot instead to give access to all bots.'",
+    "connect": "Make all your bot data just like this bot, except the prefix.",
+    "vip": "View the subscription end date."
```
Comment (FnrDev) [inline lyra/bot.json:240]:
```suggestion
    "vip": "Displays info about your subscription."
```

Comment (Raymondqx) [reply lyra/bot.json:240]: Try this one, seems better for me. 
Shows details of your subscription.

Comment (FnrDev) [reply lyra/bot.json:240]: even better, **Shows details of your current subscription.**

Comment (FnrDev) [inline lyra/bot.json:240]:
```suggestion
    "vip": "Shows details of your current subscription."
```

**Pull request merged**